app/models/organization.rb

```diff
@@ -68,6 +68,13 @@ class Organization < ApplicationRecord
   alias_attribute :old_old_username, :old_old_slug
   alias_attribute :website_url, :url
 
+  def self.simple_name_match(query)
```

Comment:
Just want to note: In the time since `/admin/organizations` was introduced, we have since added [`pg_search_scope`](https://github.com/forem/forem/blob/e69c642bf0d6076e67ac6df7426f9459edf5f804/app/models/organization.rb#L25) to the model, however that search mechanism wants to return results sorted by "best match" and does not (currently) support stem or partial-word matches the way `ILIKE %query%` does.